lib/src/components/workflowStep/workflowStepCollapsibleBody.js

```diff
@@ -1,11 +1,25 @@
-import React, { useContext, useRef } from 'react';
+import React, { useContext, useEffect, useRef, useState } from 'react';
 import { useTransition, animated } from 'react-spring';
 import * as easings from 'd3-ease';
 import { WorkflowStepContext } from './workflowStepProvider';
 
 export function WorkflowStepCollapsibleBody({ children }) {
   const { showBody } = useContext(WorkflowStepContext);
   const bodyRef = useRef(null);
+  const [shouldAllowOverflow, setShouldAllowOverflow] = useState(false);
+
+  useEffect(() => {
+    if (!showBody) {
+      setShouldAllowOverflow(false);
+    }
+  }, [showBody]);
+
+  const onTransitionEnd = () => {
+    console.log(showBody, shouldAllowOverflow);
+    if (showBody) {
+      setShouldAllowOverflow(true);
+    }
+  };
```

Comment:
Ugh. There were some overflow issues around animations and dropdowns in the body. This code just hides the overflow while the animation is in progress, then shows any overflow afterwards ready for when dropdowns appear.